tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -172,8 +173,20 @@ export class ScalarCardComponent<Downloader> {
           ...datum.metadata,
           closest: false,
           distSqToCursor: Math.hypot(
```

Comment:
Nit in naming of these properties: 

I'm not sure why the `Sq` part of the name exists. AFAICT we are calculating distance, not distance squared. Should the names instead be distanceToCursor, distanceToCursorPixels, etc...?

Reply:
I'm 100% on board with this. I was trying to avoid changing things but these names do not make sense.